aggregate_test.go

```diff
@@ -26,6 +26,46 @@ func TestAggregate(t *testing.T) {
 	}
 }
 
+func TestAggregateT(t *testing.T) {
```

Comment:
do we think that these duplicate tests add enough value? The XxxT methods are pretty much calling the Xxx equivalent and I think the `..._PanicWhenFunctionIsNotValid` tests are just enough for these. Thoughts?